### PR TITLE
Small Expansions  (Cargo+Outfit)

### PR DESCRIPTION
**Pull request by BillBobGamer**
The Expansions are to big to fit inside small intercepters.

### CODE REVIEW

File: data/human/outfits.txt [hunk 1]
```diff
@@ -291,7 +291,8 @@ outfit "Small Outfits Expansion"
 	"cooling ineffciency" .5
 	"heat dissipation" -.005
 	description "This expansion is a smaller version of the outfit expansion."
-	description "	The way that this expansion modifys the interior of your ship tends to interfere with the ship's cooling systems."
```
Comment (BillBobGamer) [inline data/human/outfits.txt]: I deleted this line by accident.

File: data/human/outfits.txt [hunk 2]
```diff
@@ -291,7 +291,8 @@ outfit "Small Outfits Expansion"
 	"cooling ineffciency" .5
 	"heat dissipation" -.005
 	description "This expansion is a smaller version of the outfit expansion."
```
Comment (BillBobGamer) [inline data/human/outfits.txt:293]: I forgot the unplunderable.

File: data/human/outfits.txt (hunk 1 above)
Comment (BillBobGamer) [reply data/human/outfits.txt]: fixed it.

File: data/human/outfits.txt (hunk 2 above)
Comment (daveeater) [reply data/human/outfits.txt:294]: Looks like a typo: "cooling ineffciency" .5
Should be "inefficiency"?

File: data/human/outfits.txt [hunk 3]
```diff
@@ -281,6 +281,19 @@ outfit "Outfits Expansion"
...
+	category "Systems"
+	thumbnail "outfit/outfit expansion"
+	cost 25000
+	"mass" 2
+	"outfit space" 7
+	"cargo space" -10
+	"cooling ineffciency" .5
+	"heat dissipation" -.005
+	description "This expansion is a smaller version of the outfit expansion."
+	description "	The way that this expansion modifys the interior of your ship tends to interfere with the ship's cooling systems.
```
Comment (daveeater) [inline data/human/outfits.txt]: "modifys" should be "modifies"

**Comment by Zitchas** [comment]
I can definitely appreciate the desire to convert *all* the space in a ship into outfit space; however this is going a bit counter to design. Part of the design intent behind many ships (especially smaller military ships such as interceptors and light warships) is to intentionally have at least a small amount of cargo space available so that they can run missions. There are a *lot* of story missions in the game that have <15 cargo, for instance. Most of these the player will simply never see or run into other problems if they are flying a ship with 0 cargo. 

This is also why many ships have odd amounts of cargo: In the lore in makes sense for them to have some cargo space, but we're all pretty well aware that combat is priority. So there's a general assumption that cargo space is going to get converted to outfit space. So we balance the amount of cargo space such that the ship will still have enough cargo to do what we think it will need to do even when optimized for combat. Not every ship is balanced this way, but many are.

As such, it is unlikely that this will be merged; although it would certainly make for a popular plug-in with many people.

**Comment by BillBobGamer** [comment]
could we make so that the small expansions are availible after the pug planet is captured?

**Comment by tehhowch** [comment]
Closing this PR; it is not going to be accepted. Making use of every last bit of space on every ship is not supposed to be possible.

As mentioned above: go ahead and [publish a plugin](https://github.com/endlessskycommunity/plugin-template) with these. Plenty of players will use it. But it isn't going to be a part of the default game.

**Comment by BillBobGamer** [comment]
ok.

**Pull request closed without merging**